app/lib/shared/platform.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:pana/pana.dart';
+
+abstract class KnownPlatforms {
+  static const String flutter = PlatformNames.flutter;
+  static const String server = PlatformNames.server;
+  static const String web = PlatformNames.web;
+  static const List<String> everywhere = const [flutter, server, web];
```

Comment:
Maybe use `all` instead of `everywhere` (I would only use `everywhere` together with a verb, e.g. `runsEverywhere`).